<ui version="4.0" >
 <class>ConfigDglAllgemeinClass</class>
 <widget class="QWidget" name="ConfigDglAllgemeinClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ConfigDglAllgemein</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>4</number>
   </property>
   <item>
    <widget class="QCheckBox" name="restoreOnStartup" >
     <property name="text" >
      <string>Erscheinungsbild beim Start wiederherstellen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Ordner mit den WÃ¶rterbuchdateien:</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>4</number>
      </property>
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="QLineEdit" name="pathToDbs" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPathToDbs" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="iconSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
